src/pages/EntityDetails/_entity-details.scss

```diff
@@ -363,29 +363,29 @@
   .entity-details__machines {
     th,
     td {
+      // Name.
       &:nth-child(1) {
         width: 15%;
       }
 
-      &:nth-child(2) {
-        width: 25%;
-      }
-
-      &:nth-child(3) {
-        width: 10%;
-      }
-
-      &:nth-child(4) {
-        width: 15%;
-      }
-
-      &:nth-child(5) {
-        width: 15%;
+      @include mobile-and-medium {
+        // AZ.
+        &:nth-child(4),
+        // Message.
+        &:nth-child(6) {
+          display: none;
+        }
       }
 
-      &:nth-child(6) {
-        max-width: 150px;
-        width: 20%;
+      @include mobile {
+        // AZ.
+        &:nth-child(4),
+        // Instance ID.
+        &:nth-child(5),
+        // Message.
+        &:nth-child(6) {
```

Comment:
In order to be consistent with how we've done it for the previous tables, maybe we can remove
```
        // AZ.
        &:nth-child(4),
```
and
```
        // Message.
        &:nth-child(6)
```
as they are already specified to be hidden in `@include mobile-and-medium`?

Reply:
Ah yes, thanks!